docs/reference/glossary.md

```diff
@@ -136,7 +136,7 @@ A BPMN element is part of a [process](#process), defining one part of its BPMN m
 
 ### Element template
 
-Use an element template to extend the Desktop Modeler or Web Modeler with domain-specific diagram [elements](#element). Element templates are used by [connectors](#connector) to create the connector-specific [element](#element) configuration.
+Use an element template to extend the [Modeler](/components/modeler/about-modeler/) with domain-specific diagram [elements](#element). The user edits such elements through a UI defined by the element template, and in the process configures standard technical bindings understood by the engine in simple and predictable ways. Element templates are used by [connectors](#connector) to create the connector-specific [element](#element) configuration.
```

Comment:
Do we want to provide a link to the bindings section?
https://docs.camunda.io/docs/next/components/best-practices/modeling/choosing-the-resource-binding-type/

Reply:
> Do we want to provide a link to the bindings section?

What would it accomplish?

Reply:
I have created a backlog ticket with https://github.com/camunda/camunda-docs/issues/7372 and assigned it to myself.